### PR TITLE
New appliance: Network Automation

### PR DESCRIPTION
**Pull request by adosztal**
Requested by David. As a test, I've built it in Docker Hub, based on my Github repo; import & boot works in my local GNS3 VM.

Don't forget to rename the container on Docker Hub from gns3_registry to network_automation. ;)

### CODE REVIEW

File: docker/network_automation/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- Ansible
+
+The /root folder is mount by default
+
+## Build and publish the Images
+
+First the base image has to be created:
+
+```
+docker build -t gns3/network_autmation:latest .
```
Comment (noplay) [inline docker/network_automation/README.md]: Their is a small typo here:
network_autmation => network_automation

**Comment by adosztal** [comment]
Whoops. Fixed, thank you.

**Comment by ehlers** [comment]
Docker image not yet available:
```
/Users/behlers# docker pull gns3/network_automation
Using default tag: latest
Error response from daemon: manifest for gns3/network_automation:latest not found
```

Docker hub https://hub.docker.com/r/gns3/network_automation/ shows no pushed image

**Comment by noplay** [comment]
I have retrigger a build

**Pull request merged**